batch-nft-reveal/app/pages/collection/[addr].tsx

```diff
@@ -0,0 +1,31 @@
+import { ethers } from 'ethers'
+import { useRouter } from 'next/router'
+import { Mint } from '../../components/collection/Mint'
+import { Layout } from '../../components/layout'
+import { RevealInfo } from '../../components/collection/RevealInfo'
+import { Gallery } from '../../components/collection/Gallery'
+
+function Collection(): JSX.Element {
+  const router = useRouter()
+  const addr = router.query.addr as string
+
+  if (addr) {
+    return (
+      <Layout>
+        <Mint contractAddr={addr}></Mint>
+        <RevealInfo contractAddr={addr}></RevealInfo>
+        <Gallery contractAddr={addr}></Gallery>
```

Comment:
Please use `Section` component for each of these and try to achieve the same look and feel as the fullstack app.